openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/base_head.html

```diff
@@ -3,9 +3,11 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
```

Comment:
I think you meant to do this in the discussion/ app, not here in the base_head.html file.

Reply:
Oh yikes yep.